ext/nnstreamer/tensor_filter/tensor_filter_tensorflow.c

```diff
@@ -99,7 +99,7 @@ tf_open (const GstTensorFilterProperties * prop, void **private_data)
 {
   int retval = tf_loadModelFile (prop, private_data);
   g_assert (retval == 0);       /** This must be called only once */
-  return 0;
+  return retval;
```

Comment:
At the current point, this modification also only returns 0 because of g_assert at 101. However, it seems to be still meaningful when we release our project compiled with G_DISABLE_ASSERT after all.

Reply:
Yes. In release, the asserts will be removed, so there should be a test for open failure in release build, to make sure that the program does not crash.